modules/kubernetes/prometheus/charts/prometheus-extras/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for prometheus-extras.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+serviceAccount:
+  name: prometheus
+
+service:
+  type: ClusterIP
+  port: 9090
+
+resources: {}
+
+remoteWrite:
+  enabled: true
+  url: ""
+  name: xenitInfra
+
+volumeClaim:
+  enabled: true
+  storageClassName: default
```

Comment:
As mentioned above, check up on how this will affect things and if it will be bound to a specific AZ

Reply:
Created anti-affinity rule, see bellow in tagged comment + managment of aks disk managment